README.md

```diff
@@ -44,7 +30,7 @@ rake db:seed
 Production credentials: In production you'll need to add production
```

Comment:
Before this line 'Production credentials can you add `Connect to the pg console:` `rails dbconsole -p`. Basically what you mentioned in slack.

Reply:
Done. I also removed this line, it didn't really make sense to me.